client/client.go

```diff
@@ -0,0 +1,126 @@
+package main
```

Comment:
I think we don't need the main package with main function and so on. Maybe let's make a `package bench` (or whatever you prefer). Keep all the necessary, utility functions/methods in `bench.go` and actual benchmarks in `bench_test.go`.

Reply:
Sure thing, I forgot about that.